core/capabilities/streams/codec.go

```diff
@@ -44,6 +43,9 @@ func (c *codec) UnwrapValid(wrapped values.Value, allowedSigners [][]byte, minRe
 				continue
 			}
 			validated[signerAddr] = struct{}{}
+			if len(validated) >= minRequiredSignatures {
```

Comment:
```suggestion
			if len(validated) == minRequiredSignatures {
```
This is more of a question - there's never a case where `len(validated)` is more than `minRequiredSignatures` given that we are iterating because we'd break as soon as equality is reached, right?

Reply:
sure